main/cicd/cicd-pipeline/config/settings/.defaults.yml

```diff
@@ -28,7 +28,7 @@ sourceRoleArn: ''
 # If this flag is set to true then you MUST also provide settings file for the staging env.
 # For example, if createStagingEnv=true and if the stgEnvName=t"prodstg" then make sure you have
 # "prodstg.yml" file in top level "config/settings" directory
-createStagingEnv: false
+createStagingEnv: true
```

Comment:
Could you change this back to false? Our default CICD deployment does not create a staging environment.